packages/@aws-cdk/aws-appmesh/test/test.virtual-gateway.ts

```diff
@@ -434,6 +434,28 @@ export = {
 
       test.done();
     },
+
+    'with shared service mesh': {
+      'Mesh Owner is the AWS account ID of the account that shared the mesh with your account'(test:Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
```

Comment:
Same comment here - let's explicitly set the account of the Stack to be different than the account of the Mesh.